game.cpp

```diff
@@ -844,29 +890,6 @@ void test_game() //!OCLINT tests may be many
     g.tick();
     assert(g.get_n_ticks() - before == 1);
   }
-  // inertia  slows down players
```

Comment:
Inertia is a useful feature. Why is this removed? Or is it moved somewhere else?